spec/defines/admin_spec.rb

```diff
@@ -40,7 +40,7 @@
 
           it do
             is_expected.to compile.with_all_deps
-            is_expected.to contain_exec('python3-django-admin help')
+            is_expected.to contain_exec('python3-django-admin migrate --noinput')
```

Comment:
Looks good. In fact, when I scroll up a few lines I can see that this is what it should have been all along.